load.go

```diff
@@ -85,15 +89,17 @@ func LoadFromPath(path string, destination interface{}, options ...CsvOptions) e
 
 // LoadFromString - Load csv from string and put it in a array of the destination's type using tags.
 // Example:
-// 	tabOfMyStruct := []MyStruct{}
-// 	err  := Load(
-// 				myString,
-// 				&tabOfMyStruct,
-// 				CsvOptions{
-// 					Separator: ';',
-// 					Header: []string{"header1", "header2", "header3"
-// 				}
-// 			})
+//
```

Comment:
my auto formatter added these, let me know if you want me to remove the extra lines

Reply:
Let's keep them if it's from the go formatter. Else, it's preferable to remove them

Reply:
yup, only have go formatter installed